iot-hub/Samples/service/RegistryManagerSample/Parameters.cs

```diff
@@ -29,7 +29,7 @@ internal class Parameters
             'd',
             "DevicePrefix",
             Required = false,
-            Default = "RegistryManagerSample-",
```

Comment:
Changing to match prefix. This was not in the delete list earlier but the sample deletes the device so this is just to guard against leaked resources. Currently the hub does not have any devices with this prefix so we don't need to worry about deleting the ones with old prefix.